packetbeat/_meta/fields.yml

```diff
@@ -1748,3 +1748,23 @@
             If the call resulted in exceptions, this field contains the exceptions in a human
             readable format.
 
+- key: tls
+  title: "TLS"
+  description: >
+    TLS-specific event fields.
+  fields:
+    - name: tls
+      type: group
+      fields:
+        - name: negotiated cypher
```

Comment:
This field name has a space in it. Probably you want an underscore here. But it looks like this file is progress because I don't see this field in the code so I'm sure you be updating all of this.